sw/device/silicon_creator/lib/boot_log.h

```diff
@@ -0,0 +1,89 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_SILICON_CREATOR_LIB_BOOT_LOG_H_
+#define OPENTITAN_SW_DEVICE_SILICON_CREATOR_LIB_BOOT_LOG_H_
+
+#include <stdint.h>
+
+#include "sw/device/lib/base/macros.h"
+#include "sw/device/silicon_creator/lib/chip_info.h"
+#include "sw/device/silicon_creator/lib/drivers/hmac.h"
+#include "sw/device/silicon_creator/lib/error.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef struct boot_log {
+  hmac_digest_t digest;
+  uint32_t identifier;
+  chip_info_scm_revision_t chip_version;
+  uint32_t rom_ext_slot;
+  uint32_t bl0_slot;
+} boot_log_t;
```

Comment:
In the original spec, I think we also wanted a ROM version here. Is there a different version number besides the scm_revision?